src/aat/resources/features/F-106 - Update Supplementary Data/F-106_Update_Supplementary_Data_C2_PRIVATE.td.json

```diff
@@ -0,0 +1,35 @@
+{
+	"_guid_": "F-106_Update_Supplementary_Data_C2_PRIVATE",
```

Comment:
As this file is only being by S-613, would be best to rename to something as "S-613_Update_Supplementary_Data_C2_PRIVATE", so it is easily identifiable.